pod-operation/hx_wheel_encodeer

```diff
@@ -0,0 +1,39 @@
+use std::thread::sleep;
+use std::time::Duration;
+use rppal::gpio::{Gpio, Level};
+
+const OUTPUT_A: u8 = 6;
+const OUTPUT_B: u8 = 7;
```

Comment:
Suggestion: calling these outputs when they are actually input pins seems a bit confusing. Consider `PIN_ENCODER_A` and `PIN_ENCODER_B`.